### PR TITLE
Add model specs.

### PR DESCRIPTION
**Pull request by hermannloose**
Added RSpec tests for all models and deleted the old Test::Unit tests accordingly. This is part of issue #15.

**Note:** Right now, both `rake` and `autotest` default to running the specs only, thus ignoring controller tests in `test/functional` etc. I didn't look into fixing this yet, as the plan is to convert all other tests to RSpec asap as well.

### CODE REVIEW

**Comment by hermannloose** [comment]
@jupp0r: Please review.

File: spec/fixtures [hunk 1]
```diff
@@ -0,0 +1 @@
+../test/fixtures/
\ No newline at end of file
```
Comment (hermannloose) [inline spec/fixtures:1]: Note: this is a (temporary) symlink, so the missing newline is git's fault.

**Comment by jupp0r** [comment]
What happened to alerting_job?

**Comment by jupp0r** [comment]
Apart from that, you have my blessing :) I really like the non-verbosity of the new tests. It's almost DSL-like reading. Can they somehow be turned into documentation?

**Comment by hermannloose** [comment]
Good catch, I restored `test/unit/alerting_job_test.rb`.

By the way, for things like that—concrete issues somewhere in the code or concerning a whole file—it's good to comment directly in the diff, thus opening a separate thread of discussion. That way, I can respond with "Done." or similar, which makes it easier to spot which issues have been addressed already and which haven't. (Also, any later changes like the commit I added are immediately reflected in the diff, so you can verify right there that I made the requested change—or spark further discussion.)

I'll have a look at RSpec regarding documentation, apart from that it already creates `rspec-results.html`, which has a nice overview of all the examples, with inlined errors, if any.

**Comment by jupp0r** [comment]
Ok, I'll comment on specific issues at their location in the diffs in the future. Regarding documentation, I thought of using the information in the rspec tests to verbally describe the behavior of a class. It would be a shame if information there would be lost or duplicated in the documentation. Anyway, I don't see this as a priority at all, just wondering.

**Pull request merged**